src/NuGetPackageVerifier/build/NuGetPackageVerifier.targets

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Target Name="VerifyPackages"
+          AfterTargets="Pack"
```

Comment:
For starters though, we could just have these export targets (that launch the exe) and make the switch to tasks later on. The composition model is the one I'm concerned about.